kbase-extension/static/kbase/js/api/StagingServiceClient.js

```diff
@@ -21,6 +21,7 @@ define(['RestAPIClient'], (RestAPIClient) => {
                 rename: { method: 'post', path: 'rename/${path}' },
                 decompress: { method: 'patch', path: 'decompress/${path}' },
                 importer_mappings: { method: 'get', path: 'importer_mappings/?${file_list}' },
+                bulkSpecification: { method: 'get', path: 'bulk_specification/?files=${files}' },
```

Comment:
This is it for setting up the client.
I've noticed there's no test suite for the staging service client or the underlying generic rest client. That could also use a refactor.

Some basic tests might go in this PR for this individual client (with the code changes), but a refactor is left for another time.